index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Porfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header-container">
+        <a href="#" class="logo">My logo</a>
+        <img src="images/icon_image/Union.png" alt="Union-btn" class="union" />
+      </nav>
+    </header>
+
+    <section class="first-container">
+      <h1 class="intro-text">
+        I'm Kwamay. <br />
+        Glad to see <br />you!
+      </h1>
+      <p class="intro-msg">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h3 class="connect">LET'S CONNECT</h3>
+      <ul class="social">
+        <li>
+            <a href="#" class="icons-link"></a>
+          <img
+            src="images/icon_image/twitter.png"
+            alt="twitter"
+            width="20"
+            height="20"
+          />
+        </li>
+        <li>
+            <a href="#" class="icons-link"></a>
+          <img
+            src="images/icon_image/linkedin.png"
+            alt="linkedin"
+            width="20"
+            height="20"
+          />
+        </li>
+        <li>
+            <a href="#" class="icons-link"></a>
+          <img
+            src="images/icon_image/medium.png"
+            alt="medium"
+            width="20"
+            height="20"
+          />
+        </li>
+        <li>
+           <a href="#" class="icons-link">
+         <img
+          src="images/icon_image/github.png"
+          alt="github"
+          width="20"
+          height="20"
+        />
+       </li>
+        <li>
+            <a href="#" class="icons-link">
+          <img
+            src="images/icon_image/angelist.png"
+            alt="github"
+            width="20"
+            height="20"
+          />
+        </li>
+      </ul>
```

Comment:
- [ ] Great job on implementing the changes suggested by the reviews reviewer, However, some of the changes were not implemented completely. To fix this issue kindly consider the suggestions below:
   - [ ] Kindly make sure that each `icon` is wrapped in an `a` tag as mentioned by the previous reviewer.  This will provide a better user experience by allowing users to easily connect with you through your social media handles. By associating each `icon` with an `a` tag, it conveys the intention of linking to your social media profiles. This way, users can click on the icons and be directed to the corresponding social media platforms.
   
 ### Example:
 ```
  <ul>
   <li>
     <a>
      <img src="icon_link" alt="alt_text" />
    </a>
   </li>
  </ul>
```